tests/spark/test_spark_model_export.py

```diff
@@ -160,15 +160,15 @@ def spark_model_estimator(iris_df, spark_context):
 
 
 @pytest.fixture
-def model_path(tmpdir):
-    return str(tmpdir.mkdir("model"))
+def model_path(tmp_path):
+    return os.path.join(tmp_path, "model")
```

Comment:
Will this directory exist directly?

Reply:
Good catch, but `save_model` should work when the directory doesn't exist.